redhat-cop/infra-ansible
# Update IDM User Management

### What does this PR do?
This PR does some cleanup of the user-management role for IDM. 

- Currently there is a symlink that pointed to nothing. 

- There was also an issue with the manage-users role being nested under user-management, so the current test inventory wasn't working as expected.

- The last issue was that the property to set password expiration was listed as `expiration_date`, where it should actually be `krbpasswordexpiration`.

** At this point, it should be noted that we still can't use the krbpasswordexpiration during user creation because of an existing bug. PR has been submitted to ansible to resolve: https://github.com/ansible/ansible/pull/42052 . Once this has been resolved, we can follow up with an additional PR to enable this functionality.

### How should this be tested?
Run the user-management tests against an existing IDM server. You should see the users be able to be created and modified.

### Is there a relevant Issue open for this?
No

### People to notify
cc: @redhat-cop/infra-ansible @oybed @logandonley @makentenza
